yetanotherco/aligned_layer
# chore (operator): separate minting & depositing into strategy

- Created new script `scripts/mint_mock_token.sh` to mint mock token
- Created make target to mint mock token
- Modified deposit into strategy script to take in strategyAddress & not try to mint token

To test:
you can run `make operator-full-registration` and see that it works or
run step by step:
1. `make operator-get-eth`
2. `make operator-register-with-eigen-layer`
3. `make operator-mint-mock-tokens`
4. `make operator-deposit-into-mock-strategy`
